cmd/kubeadm/app/cmd/join.go

```diff
@@ -267,6 +269,10 @@ func addJoinConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiv1beta2.JoinConfig
 
 // addJoinOtherFlags adds join flags that are not bound to a configuration file to the given flagset
 func addJoinOtherFlags(flagSet *flag.FlagSet, joinOptions *joinOptions) {
+	flagSet.BoolVarP(
+		&joinOptions.forceJoin, options.ForceJoin, "f", false,
+		"Skip check for existing Node in the cluster.",
```

Comment:
as explained here, we cannot do that:
https://github.com/kubernetes/kubernetes/issues/89512#issuecomment-604554642

this bug however needs to be fixed in immutable upgrades:
https://github.com/kubernetes/kubernetes/issues/89501